src/server/middlewares/index.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, Response, NextFunction } from 'express'
+import jwt from 'jsonwebtoken'
+import dotenv from 'dotenv'
+import path from 'path'
+dotenv.config({
+  path: path.resolve('./prisma/.env'),
+})
+type DecodedData = {
+  userId: number
+}
+export function tokenVerifier() {
```

Comment:
@Agrajak 별건 아니지만 포매팅 관련된건데요 코드 컨텍스트가 달라질 때 줄 바꿈을 넣어주는 게 더 깔끔하다고 생각해요!

그리고 `prisma/.env` 파일은 프리즈마와 관련된 환경 변수를 저장하는 곳이고 그 이외의 환경 변수는 프로젝트 루트 디렉토리의 `.env`에 기록하는게 맞을거같습니당

```suggestion
import path from 'path'

dotenv.config()

type DecodedData = {
  userId: number
}

export function tokenVerifier() {
```

Reply:
동의합니다!!

Reply:
네 그게 맞겠네요!